core/common/src/rpc/web/WebAppRpcProtocol.ts

```diff
@@ -122,6 +122,8 @@ export abstract class WebAppRpcProtocol extends RpcProtocol {
       case 502: return RpcRequestStatus.BadGateway;
       case 503: return RpcRequestStatus.ServiceUnavailable;
       case 504: return RpcRequestStatus.GatewayTimeout;
+      case 408: return RpcRequestStatus.RequestTimeout;
+      case 429: return RpcRequestStatus.TooManyRequests;
```

Comment:
What's our retry time for requests? If we're already sending too many do we want to try and do a back-off instead of continually spamming the server with more requests?

Reply:
I think the below function is currently determining how long to wait before retrying a transient error.

protected computeRetryAfter(attempts: number) {
    return (((Math.pow(2, attempts) - 1) / 2) * 500) + 500;
  }

I don't think this is really that fast... When I was trying to solve the failure to receive data from the peer error for a user, I ended up going to 10 retries in 2.x, and a maximum wait time of 5 seconds. 

https://github.com/iTwin/itwinjs-core/commit/bfe3d0182a79892584145aaf8392c090c906e061#diff-1ea45b8ea96d2aaf1d5d3c0da5abb5d1c4c00621dc016da2bb80a0e9783a1b2d 

Since we're trying to take advantage of transient errors to get frontends to retry requests, 1-5 seconds sort of seems fast  when we consider that we've seen some RPC operations take ~6 minutes... But I don't know if 1-5 seconds is necessarily long in the context of an actual transient error.

Reply:
Any more thoughts here? DR is reporting more 504s exceeded transient fault limits. Bumping to 4 won't speed things up, but may stop us from getting exceeded transient fault limit.

Reply:
that's fine with me. This isn't really a fix more of a bandaid though.